atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -58,13 +58,13 @@ export const crashGridTableColumns = {
     label_table: "Collision Description",
     type: "String",
   },
-//   "units { unit_desc_lkp { label } }": {
-//     searchable: false,
-//     sortable: false,
-//     label_table: "Unit Description",
-//     type: "String",
-//     hidden: true,
-//   },
+  "units { unit_desc_lkp { label } }": {
+    searchable: false,
+    sortable: false,
+    label_table: "Unit Description",
+    type: "String",
+    hidden: true,
```

Comment:
why is this hidden?

Reply:
it is hidden because we don't render it in directly in the UI, but it it is used for the charts on the location details page. i had i had forgotten that i commented it out completely and it in fact did break the location details page, we just didn't notice 🙃